test/index.test.js

```diff
@@ -88,16 +89,45 @@ describe('Out of Band cache', () => {
       assume(calledRefresh).is.truthy();
     });
 
+    it('can override the cache-level maxAge', async function () {
+      const fridge = new Cache({ maxAge: -10, maxStaleness: -10 }); // items are immediately stale
+      await fridge.get('good milk', { maxAge: 1000 }, () => Promise.resolve('milk'));
+      const milk = await fridge.get('good milk', { }, () => () => Promise.reject(new Error('gross')));
```

Comment:
I think you nested 1 too many times in this predicate, you're returning a function, not a promise.

```js
const milk = await fridge.get('good milk', { }, async () => { new Error('gross') } );
```

Reply:
lol yeah. I did.